client/blocks/reader-related-card-v2/index.jsx

```diff
@@ -26,17 +26,18 @@ const RELATED_IMAGE_WIDTH = 385; // usual width of featured images in related po
 function AuthorAndSiteFollow( { post, site, onSiteClick, followSource } ) {
 	const siteUrl = getStreamUrl( post.feed_ID, post.site_ID );
 	const siteName = ( site && site.title ) || post.site_name;
-	const authorAndSiteAreDifferent = ! areEqualIgnoringWhitespaceAndCase( siteName, post.author.name );
+	const authorName = get( post, 'author.name', '' );
```

Comment:
What is this fixing? Is there an issue with the way `areEqualIgnoringWhitespaceAndCase` functions?